saml_idp.gemspec

```diff
@@ -41,6 +41,7 @@ defaults in a Production environment. Post any issues you to github.
   s.add_dependency('builder')
   s.add_dependency('httparty')
   s.add_dependency('nokogiri')
+  s.add_dependency('xmlenc', '>= 0.6.4')
```

Comment:
One concern is that the encryption stuff isn't necessary in previous versions and I tend to be sensitive to adding new dependencies (while this one seems entirely worth it in the case where users care to have it encrypted). What's your feel on how necessary or common it is in the SAML community to use what you're introducing? If its not ~80% of SAML users using this technique, perhaps we should make this development dependency and test the libraries presence when we try to run this encryption work you've added.

Not sure I'm making sense, if not I can illustrate a bit with an example.